corehq/apps/data_cleaning/views/main.py

```diff
@@ -1,13 +1,18 @@
+from datetime import datetime
 from memoized import memoized
 
 from django.contrib import messages
+from django.http import HttpResponseRedirect
```

Comment:
@orangejenny can you use the `redirect` shortcut below? It's weird to have both `HttpResponseRedirect` and `redirect` in the same set of imports. we are already using `redirect`